src/granite/table.cr

```diff
@@ -53,6 +53,10 @@ module Granite::Table
       @@primary_name
     end
 
+    def self.primary_type
+      @@primary_type
+    end
```

Comment:
There is no such class variable.

Reply:
There was a macro variable assignment to primary_type = PRIMARY[:type]. It was not used anywhere so i made the method. The class variable was not declared and i will add it. Thanks for noticing